### PR TITLE
account -> assessments relationship support [#140766113]

### PR DESCRIPTION
**Pull request by thematthopkins**

### CODE REVIEW

**Comment by thematthopkins** [comment]
I think this change should be backwards compatible.

`impact-app` tests pass while pointing to this branch.  Going to run through `impact-manage` to ensure it's ok too.

File: addon/models/account.js [hunk 1]
```diff
@@ -63,6 +63,7 @@ export default Model.extend({
   affiliationsCertifications: attr('string'),
   primaryCustomerTarget: attr('string'),
   assessment: belongsTo('assessment', { async: true }),
+  assessments: hasMany('assessment', { async: true, inverse: 'account' }),
```
Comment (jonpitch) [inline addon/models/account.js]: @thematthopkins remind me again why this was necessary?

**Comment by thematthopkins** [comment]
`impact-manage` also appears to be backwards compatible - all tests passing.

File: addon/models/account.js (hunk 1 above)
Comment (thematthopkins) [inline addon/models/account.js]: Without it, only one assessment could exists in the ember store at a time, even though they would get included in the Json API responses.  This fixed it - I think Ember was seeing just the assessment relationship, and deducing that only one assessment could exist per account.

Comment (jonpitch) [inline addon/models/account.js]: So what happens if you remove the `assessment` relationship? Is it a big refactor?

Comment (thematthopkins) [inline addon/models/account.js]: Right now, we still depend on `assessment` acting as the _active_ assessment.  I think you're right and it should probably be renamed to `activeAssessment`.  That's potentially going to be a pain.

If only we had static types w/ a nifty ctrl+r = rename.

Is it worth doing this now?  I think this is a good place for the active assessment relationship to live, but maybe worth discussing.

Comment (jonpitch) [inline addon/models/account.js]: @thematthopkins gotcha, so let's leave as is for now, but maybe leave a comment that `assessment` is really the `active`? We'll have to do some deprecations over time to rename these things.

**Comment by jonpitch** [comment]
@thematthopkins looks good, let's coordinate with @zkrzyzanowski so we can roll out a few of these PRs, #48 and #46 at the same time.

**Pull request merged**